Cubical/Data/NatMinusOne/Base.agda

```diff
@@ -0,0 +1,21 @@
+{-# OPTIONS --cubical --no-exact-split --safe #-}
+module Cubical.Data.NatMinusOne.Base where
+
+open import Cubical.Core.Primitives
+open import Cubical.Data.Nat
+
+data ℕ₋₁ : Set where
+  neg1 : ℕ₋₁
+  suc  : (n : ℕ₋₁) → ℕ₋₁
+
+_+₋₁_ : ℕ → ℕ₋₁ → ℕ₋₁
+0 +₋₁ n = n
+suc m +₋₁ n = suc (m +₋₁ n)
+
+ℕ→ℕ₋₁ : ℕ → ℕ₋₁
+ℕ→ℕ₋₁ n = n +₋₁ suc neg1
+
+1+_ : ℕ₋₁ → ℕ
+1+ neg1 = zero
+1+ suc n = suc (1+ n)
+
```

Comment:
For completeness, also add the following function? It's also a clearer way to write `n +₋₁ neg1`.
```agda
_-1 : ℕ → ℕ₋₁
zero -1 = neg1
(suc n) -1 = suc (n -1)
```